.github/workflows/create-deb.yaml

```diff
@@ -0,0 +1,34 @@
+name: create-deb
+
+on:
+  release:
+    types: [released]
```

Comment:
Now this works, let's move this to `release.yaml` and remove this file?

Reply:
Yeah, unless you feel the `release.yaml` file would get messy, and there is a way for the `release.yaml` file to call upon this `create-deb.yaml` file as a step. I don't know how this works but in the future we might want to use [`build-rpm-action`](https://github.com/marketplace/actions/build-rpm-action) to build `.rpm` packages for Fedora and Fedora-based Linux distributions, and packages for macOS.

Reply:
Yes, I think we should try and consolidate them unless there's a reason to have them be separate. 

We have different ones for `test-python` etc, because we want to run those on changes to the `prql-python` path. but that's not required here.

Reply:
Alright